lib/Parse/ParsedSyntaxNodes.cpp.gyb

```diff
@@ -27,15 +27,15 @@ using namespace swift::syntax;
 %   for child in node.children:
 %     if child.is_optional:
 Optional<Parsed${child.type_name}>
-Parsed${node.name}::getDeferred${child.name}() {
-  auto &RawChild = getRaw().getDeferredChildren()[${node.name}::Cursor::${child.name}];
+Parsed${node.name}::getDeferred${child.name}(const SyntaxParsingContext *SyntaxContext) {
+  auto RawChild = getRaw().getDeferredChild(${node.name}::Cursor::${child.name}, SyntaxContext);
   if (RawChild.isNull())
     return None;
   return Parsed${child.type_name} {RawChild.copyDeferred()};
 }
 %     else:
-Parsed${child.type_name} Parsed${node.name}::getDeferred${child.name}() {
-  return Parsed${child.type_name} {getRaw().getDeferredChildren()[${node.name}::Cursor::${child.name}].copyDeferred()};
+Parsed${child.type_name} Parsed${node.name}::getDeferred${child.name}(const SyntaxParsingContext *SyntaxContext) {
+  return Parsed${child.type_name} {getRaw().getDeferredChild(${node.name}::Cursor::${child.name}, SyntaxContext).copyDeferred()};
```

Comment:
`copyDeferred()` is not needed I believe

Reply:
It’s still used in one place, I’ll remove it in https://github.com/apple/swift/pull/36362/commits/81f7d1e0d7ef81cbcb21a9b757eb664e27bb42a9 which is part of https://github.com/apple/swift/pull/36362